libraries/AP_HAL_ChibiOS/hwdef/common/stm32_util.c

```diff
@@ -291,6 +291,10 @@ void peripheral_power_enable(void)
     // others need it active high
     palWriteLine(HAL_GPIO_PIN_VDD_3V3_SD_CARD_EN, 1);
 #endif
+    for (i=0; i<20; i++) {
```

Comment:
As a general question why use this loop structure? I can't seem to find any justification for it quickly via google, and I've noticed you doing this a couple of times now.

Reply:
I have deliberately avoided being dependent on CH_CFG_INTERVALS_SIZE being 32. It can be 16 in ChibiOS. As we use a 1MHz clock that means this function is limited to 65ms in that case. 
These are the only places we use chThdSleep() with a value that could take us past 65ms, thus the loops